test-case/multiple-pause-resume.sh

```diff
@@ -56,7 +56,7 @@ rnd_range=$((rnd_max - rnd_min))
 [[ $rnd_range -le 0 ]] && dlogw "Error random range scope [ min:$rnd_min - max:$rnd_max ]" && exit 2
 
 tplg=${OPT_VAL['t']}
-func_pipeline_export "$tplg" "type:any"
+func_pipeline_export "$tplg" "type:any & ~pcm:Amplifier Reference"
```

Comment:
I would go one step further and remove 'Amplifier'.

We've seen cases where we talk about 'Echo Reference'.

Reply:
@plbossart , so you mean that we should remove both smart amplifier pipeline and echo reference pipeline form the test?

Reply:
"amplifier reference' and 'echo reference' are the same thing, aren't they?

Reply:
I meant use ~pcm:Reference

Reply:
here is the output of these 2 pipeline parsed by sof-tplgreader.py:
```
pcm=Speaker;id=2;type=playback;cap_name=Smart Amplifier Playback 2;fmts=S16_LE S24_LE S32_LE;fmt=S16_LE;rate_min=48000;rate_max=48000;ch_min=2;ch_max=2;rate=48000;channel=2;dev=hw:0,2;snd=/dev/snd/pcmC0D2p;smart_amp=SMART_AMP3.0;
```

```
pcm=Amplifier Reference;id=3;type=capture;cap_name=Echo Reference Capture 3;fmts=S16_LE S24_LE S32_LE;fmt=S16_LE;rate_min=48000;rate_max=48000;ch_min=2;ch_max=2;rate=48000;channel=2;dev=hw:0,3;snd=/dev/snd/pcmC0D3c;smart_amp=SMART_AMP3.0;
```

if we want to filter them all, we can use `~smart_amp`